config/locales/de.yml

```diff
@@ -41,6 +41,10 @@ de:
     index:
       lead: "Wir bieten unglaubliche Lösungen für vielfältige Use Cases."
 
+  user-dashboard:
+    drop-down:
+      dashboard: "My Dashboard"
```

Comment:
German please.

Reply:
Is it ok to change it to "Mein Dashboard"?
"Instrumententafel" and "Armaturenbrett" sound weird in this context.

Reply:
Haha sure.

Reply:
@zhamanakov ?